src/_includes/layouts/base.njk

```diff
@@ -32,13 +32,33 @@
     <meta name="twitter:image" content="{{ cover }}">
     <meta name="twitter:creator" content="@geteslint">
 
-
-    <script type="module">
+    <script>
         // This is a capable browser, let's improve the UI further!
         document.documentElement.classList.add("enhanced");
         document.documentElement.classList.remove('no-js');
     </script>
 
+    <style>
```

Comment:
The purpose of this style block is to avoid layout shifts.

1.  `.no-js #nav-toggle { display: none !important }` - Since the toggle button is [not hidden by default](https://github.com/eslint/playground/pull/64/files/efd44450852b8a85c918886ed6e405cbee84054c#r889601233) need to hide it when js is disabled (`no-js`).

2.   `@media all and (min-width: 680px) { .enhanced #nav-toggle{ display: none;  } }` - since the toggle button is [not hidden by default](https://github.com/eslint/playground/pull/64/files/efd44450852b8a85c918886ed6e405cbee84054c#r889601233)  need to hide it in the mobile view (`enhanced`). 


3.  ` @media all and (max-width: 679px) { .no-js #nav-panel {   display: block !important; }                                            .enhanced #nav-panel{ display: none; } }`. 
     -  To open the nav panel by default when js is disabled in the mobile view (`no-js`).
     -  To hide the nav panel by default in the mobile view (`enhanced`).

Reply:
Can you include a comment explaining this?